test/output/slim/block_admonition.html

```diff
@@ -0,0 +1,72 @@
+<!-- .basic -->
+<div class="admonitionblock warning">
+  <table>
+    <tr>
+      <td class="icon">
+        <div class="title">Warning</div>
+      </td>
+      <td class="content">Watch out for dragons!</td>
+    </tr>
+  </table>
+</div>
+
+<!-- .basic_multiline -->
```

Comment:
Ah, and also I’ve dropped `block_` prefix in beta.3, to be consistent with Asciidoctor (IIRC `block_` prefix in node names is a legacy).